### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vladgusakovskiy**
[DEMO LINK](https://vladgusakovskiy.github.io/react_dynamic-list-of-todos/)

### CODE REVIEW

File: src/components/TodoModal/TodoModal.tsx [hunk 1]
```diff
@@ -15,30 +39,37 @@ export const TodoModal: React.FC = () => {
...
-              <strong className="has-text-danger">Planned</strong>
+              {
+                completed
+                  ? <strong className="has-text-success">Done</strong>
+                  : <strong className="has-text-danger">Planned</strong>
+              }
 
               {' by '}
 
               <a href="mailto:Sincere@april.biz">
```
Comment (SergeyPistelev) [inline src/components/TodoModal/TodoModal.tsx:71]:
```suggestion
              <a href={user?.email}>
```

**Review by SergeyPistelev — APPROVED** [review]
Well done!)
Add specific user's email to todoModal.

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  selectedTodo: Todo | undefined;
```
Comment (AndriiRut) [inline src/components/TodoList/TodoList.tsx:7]:
```suggestion
  selectedTodo: Todo | null;
```

Comment (AndriiRut) [reply src/components/TodoList/TodoList.tsx:7]: if we possibly have an object we can replace it bu null!

**Review by AndriiRut — APPROVED** [review]

File: src/components/TodoList/TodoList.tsx (hunk 2 above)
Comment (AndriiRut) [reply src/components/TodoList/TodoList.tsx:7]: good job 🚀

**Pull request closed without merging**